README.md

```diff
@@ -5,37 +5,87 @@ pymtbl: Python bindings for the mtbl sorted string table library
 [mtbl](https://github.com/farsightsec/mtbl)'s reader, writer, sorter, and
 merger interfaces. The `examples/` directory contains scripts demonstrating
 each of these interfaces. The following transcript shows the basic reader and
-writer interfaces:
+writer interfaces (nb. the traceback is expected behavior):
 
     >>> import mtbl
     >>> w = mtbl.writer('example.mtbl', compression=mtbl.COMPRESSION_SNAPPY)
-    >>> w['key1'] = 'val1'
-    >>> w['key17'] = 'val17'
-    >>> w['key2'] = 'val2'
-    >>> w['key23'] = 'val23'
-    >>> w['key3'] = 'val3'
-    >>> w['key4'] = 'val4'
-    >>> w['key05'] = 'val05'
+    >>> w[b'key1'] = b'val1'
```

Comment:
This is the right thing to do, although the interface is going to be ugly. I don't suppose that we can accept both? Perhaps accept `object` and then write a function that decodes or raises a `ValueError` as appropriate.

We should not do this with `varint_decode`, though. That really should only work with `bytes`.

Reply:
It would be nicer to allow strings and bytes in Python3, but I think only allowing bytes for now is okay since Jeff documented this change pretty well in the README when he did the port.